app/learning/index.md

```diff
@@ -39,7 +39,7 @@ Then install the needed generator(s). Generators are npm packages named `generat
 npm install -g generator-webapp
 ```
 
-New Node and npm users might runs into permissions issues. These issues shows up in the form of `EACCESS` errors during installation. Refer to the [npm guide to fix permissions]
+New Node and npm users might run into permissions issues. These issues show up in the form of `EACCESS` errors during installation. Refer to the [npm guide to fix permissions]
```

Comment:
Hi,
Because we're saying "issues" which is **_plural_**, we then need to say "show". 
If, however, we said "this issue" - **_singular_** - then we would say "shows".
Make sense?
Kev